hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFileOps.java

```diff
@@ -73,8 +74,8 @@ public void setupOzoneFileSystem()
           throws IOException, TimeoutException, InterruptedException {
     OzoneConfiguration conf = new OzoneConfiguration();
     conf.setInt(FS_TRASH_INTERVAL_KEY, 1);
-    conf.set(OMConfigKeys.OZONE_OM_LAYOUT_VERSION, "V1");
-    conf.setBoolean(OMConfigKeys.OZONE_OM_ENABLE_FILESYSTEM_PATHS, false);
+    OzoneTestUtils.configureFSOptimizedPaths(conf,
+            true, OMConfigKeys.OZONE_OM_LAYOUT_VERSION_V1);
```

Comment:
Original OZONE_OM_ENABLE_FILESYSTEM_PATHS is false, but now is expected to pass true?

Reply:
Yes, this test class is to verify HDDS-2939 cases. With this PR, we are ensuring that FSO(FileSystemOptimized) feature will be enabled if both flags are TRUE.